diff --git a/Source/NSTextView.m b/Source/NSTextView.m
index 8d98d1334..f137fdfbd 100644
--- a/Source/NSTextView.m
+++ b/Source/NSTextView.m
@@ -64,7 +64,6 @@
 #import <Foundation/NSTimer.h>
 #import <Foundation/NSUndoManager.h>
 #import <Foundation/NSValue.h>
-#import <Foundation/NSAutoreleasePool.h>
 
 #import "AppKit/NSApplication.h"
 #import "AppKit/NSAttributedString.h"
@@ -1171,9 +1170,6 @@ to this method from the text container or layout manager.
 */
 - (void) setTextContainer: (NSTextContainer *)container
 {
-
-  ENTER_POOL
-  
   NSUInteger i, c;
   NSArray *tcs;
   NSTextView *other;
@@ -1237,8 +1233,6 @@ to this method from the text container or layout manager.
   _currentInsertionPointMovementDirection = 0;
 
   [self _updateMultipleTextViews];
-  
-  LEAVE_POOL
 }
 
 - (void) replaceTextContainer: (NSTextContainer *)newContainer
@@ -4046,10 +4040,7 @@ Figure out how the additional layout stuff is supposed to work.
     }
 
   // Remove any existing tooltips in the redrawn rectangle.
-  if (_rFlags.has_tooltips != 0)
-  {
-    [[GSToolTips tipsForView: self] removeToolTipsInRect: rect];
-  }
+  [[GSToolTips tipsForView: self] removeToolTipsInRect: rect];
   {
     NSRange r;
     NSUInteger i = drawnRange.location;
