diff --git a/Source/NSTextField.m b/Source/NSTextField.m
index 947e7915e..0235fd9ee 100644
--- a/Source/NSTextField.m
+++ b/Source/NSTextField.m
@@ -120,26 +120,6 @@ static Class textFieldCellClass;
   return YES;
 }
 
-- (void) setPlaceholderString: (NSString *)string
-{
-  [_cell setPlaceholderString: string];
-}
-
-- (NSString *) placeholderString
-{
-  return [_cell placeholderString];
-}
-
-- (void) setPlaceholderAttributedString: (NSAttributedString *)string
-{
-  [_cell setPlaceholderAttributedString: string];
-}
-
-- (NSAttributedString *) placeholderAttributedString
-{
-  return [_cell placeholderAttributedString];
-}
-
 /** <p>Returns whether the NSTextField is editable. By default a NSTextField
     is not editable.</p>
     <p>See Also: -setEditable: [NSCell-isEditable]</p>
